### PR TITLE
feat: [MC-690] Distinct Pocket dev JWK key ids

### PR DESCRIPTION
**Pull request by mmiermans**
## Goal
Use different jwk keys in development and production for the Content team's Pocket JWKs.

### What changed
- Feat: Change pocket development key ids (kids).
- Fix: Get Github superlinter to pass.
- Fix: Add `--elv2-license accept` to start:dev script.
- Fix: Set `NODE_ENV` to 'development' in Pocket-Dev. Previously it was set to 'production'. :confused: 

## Implementation Decisions
- https://getpocket.dev/.well-known/jwk throws an SSL error, and dotcom-gateway doesn't have a dev environment set up, so we decided to distinguish based on kid.

## References

JIRA ticket:
* https://mozilla-hub.atlassian.net/browse/MC-690

dotcom-gateway PR:
* https://github.com/Pocket/dotcom-gateway/pull/146

### CODE REVIEW

**Review by jpetto — APPROVED** [review]

**Comment by katerinachinnappan** [comment]
@mmiermans prob good idea to update the README for running `start:dev` when running the api locally

**Review by katerinachinnappan — APPROVED** [review]
🔥

File: .aws/src/main.ts [hunk 1]
```diff
@@ -147,7 +147,7 @@ class AdminAPI extends TerraformStack {
           ],
           envVars: [
             {
-              name: 'ENVIRONMENT',
```
Comment (mmiermans) [inline .aws/src/main.ts:150]: I checked that `ENVIRONMENT` was not used by searching the codebase.

File: src/jwtUtils.ts [hunk 2]
```diff
@@ -124,7 +124,7 @@ function getJwksClient(jwksUri: string) {
   return jwksClient({
     jwksUri,
     cache: true, // Default Value
-    cacheMaxEntries: 5, // Default value
```
Comment (mmiermans) [inline src/jwtUtils.ts:127]: We increased the number of keys. Increasing `cacheMaxEntries` might prevent performance issues, if we have more than 5 keys. The number 100 was chosen arbitrarily.

**Pull request merged**